styles.css

```diff
@@ -469,3 +469,108 @@ h1 {
   border: solid 1px;
   background-color: white;
 }
+
+.contact-form {
+  padding: 114px 24px;
+  background-image: url(./images/contact\ form\ shapes-mobile@2x.png);
+  min-height: 800px;
+  background-repeat: no-repeat;
+  background-size: 63% 83%;
+  background-position: right 30px;
+  border-top-left-radius: 5rem;
+  background-color: #6070ff;
+  margin-top: -4rem;
+  display: grid;
+  grid-template-rows: auto auto;
+}
```

Comment:
- [ ] I would kindly suggest you add the background image to the contact section of your project as shown in the Figma design, as currently, your design is missing the background image. I notice there is a background image property in your design, i would advice you target the right directory of the image so that the image can appear in your project

Your Design 👇 

![lanz 1](https://user-images.githubusercontent.com/78634841/212946750-a40de437-46c1-448b-a761-c589327749f5.png)

Figma design 👇 

![cont-mob](https://user-images.githubusercontent.com/78634841/212947776-593f1e7e-c9cd-44c5-b4b5-189e894395a0.png)